kubernetes/apps_kubeclient.go

```diff
@@ -743,7 +758,7 @@ func (kc *kubeClient) getPods(namespace string, uid types.UID) ([]v1.Pod, error)
 		return nil, err
 	}
 
-	var appPods []v1.Pod
+	appPods := make([]v1.Pod, 0)
```

Comment:
you might as well just do `appPods := []v1.Pod{}`

Reply:
It's a good point, and I blame my inexperience with Go.  It's too late for this PR, but I am creating one that addresses the comments you made in the previous PR (for the /apps endpoint) and I'll change it there.